migration/migrator/migrations/system/20191027013443_separate_shipper_dirs.py

```diff
@@ -0,0 +1,11 @@
+import os
+import shutil
+
+IN_PROGRESS_PATH = '/var/local/submitty/grading'
+
+
+def up(config):
+    if not os.path.exists(IN_PROGRESS_PATH):
+        os.mkdir(IN_PROGRESS_PATH)
+    shutil.chown(IN_PROGRESS_PATH, 'submitty_daemon', 'submitty_daemonphp')
+    os.chmod(IN_PROGRESS_PATH, 0o770)
```

Comment:
Even if you aren't going to implement a down migration, add one which just passes (does nothing).